1906-10-08.xml

```diff
@@ -1353,9 +1349,1022 @@
+                        scaleded everyone in the engine room, which after the explosion was a wreck. </p>
+                    <p>Mr. Kendall, the chief engineer, was quite equal to the occasion and managed
+                        to keep the steamer moving slowly ahead with the remaining egine and
+                        propeller, but the vessel was making water fast, and the after part of the
+                        ship commenced slowly settling dow in the water. Captain Watkins, who was
+                        quite alive to the situation, decided to steer direcr for the Island of
+                        Zante, where the steamer arrived about dark. In the meantime pumping was
+                        going on incesstantly, but the in rush of the water was gaining on the pump.
+                        Once inside the harbour of Zante the pumps were in a better position to work
+                        and therfore more efficaceous, and in this way they were able to keep down
+                        the leakage.</p>
+                    <p>Captain Watking at once wired for the assistance on board and on Wednesday
+                        afternoon at 5 p.m. H.M.S Suffolk arrived from Malta.</p>
+                    <p>The commander at once boarded the Isis accopanied by his chief-engieer and
+                        chief carpenter, and shortly afterwards a large pentoon containing divers;
+                        apparatus, a staff of blue jackets and four expert divers came alongside and
+                        at once commenced operations assisted by the officers and engineers of the
+                        Isis. They worked the whole of Wednesday night by the aid of electric light,
+                        enabling the divers to see under the water. Operations were sucessfully
+                        terminated on Thursday at 9 a.m.</p>
+                    <p>Before the arrival of H.M.S Suffolk, the first officer, Mr. Cyril Brook, the
+                        second officer, Mr. D.J Cornwall Jones, and the fourth officer, Mr. A.N
+                        Gannery, successfully carried out the commander's plane for ascertaining the
+                        damage, the two latter diving down under the ship's stern, where they
+                        eventually succeeded in inserting a wooden plug of very large dimensions
+                        into the hole caused by the breaking off of the propeller. Great praise is
+                        due to these young officers, who voluntarily undertook this arduous
+                        enterprise which was in reality the work of a trained and expert diver, and
+                        the divers themselves expressed their admiration and suprise that such a
+                        piece of work could have been done without the aid of a diving
+                        apparatus.</p>
+             </div>   
+                <cb n="5"/>
```

Comment:
When a column break comes in the middle of an article, you don't need to cut the article into two divs. You can just put the cb in the middle of the article, wherever it falls in the text.